docs/docs/licensing.md

```diff
@@ -19,13 +19,15 @@ The current limitations are applied to the community version:
 
 ## Licensed
 
-The exact model for buying Qlik Core will be announced late spring/early summer, and will require
+The exact model for buying Qlik Core will be announced during summer, and will require
 a side-car license service to unlock the Qlik Associative Engine service. See the
 [Licenses service documentation](./services/licenses.md) for additional information and how you can
 prepare your solution for Qlik Core licensing.
 
-!!! Note "Configuring licensing"
-    You need to pass in a setting to Qlik Associative Engine to enable licensing.
+!!! Note "Getting a beta license"
+    During the beta you can get a license by filling in the form [here](./licensing-form.md).
```

Comment:
`Filling out the form`.

Only linking a small word like `here` feels a bit hidden, perhaps linking the full "action" segment of the sentence, `filling out this form`?